credible/credible-introduction.mdx

```diff
@@ -238,7 +238,7 @@ Explore more use cases in the [Assertions Book](/assertions-book/assertions-book
 
 ## Next Steps
 
-Ready to get started with the Phylax Credible Layer? Explore these resources:
+Want to try the Phylax Credible Layer locally? Explore these resources:
```

Comment:
This doesn't add any value or clarity. If it's running locally, it's still the Credible Layer, so why change it to be specifically local?